standards/practices/testing.md

```diff
@@ -0,0 +1,198 @@
+# Testing
+## Summary
+We test our code with as few tests as possible to provide reasonable confidence our software functions as intended.
```

Comment:
I don't like the negative emphasis of "as few tests as possible" and suggest saying something like "We test our code with sufficient, while no more than necessary, tests to provide adequate confidence our software functions as intended"

Reply:
I love the emphasis of "as few tests as possible," but understanding softening it a bit. I think the perspective is we aren't putting people in outer space or controlling robotic arms doing brain surgery.

Reply:
I don't see it as negative at all. I see it as minimalist and therefore freeing. We're not saying, "don't write tests," we're saying "write as many tests as required to provide confidence, but no more."

We're also saying the same thing. I prefer the way it's written now b/c it flows better so I don't have to slow down and reread it to understand the meaning. I'm open to rephrasing it, but not at the cost of reading experience.

Reply:
I think we are saying the same thing but the current form leaves me with the impression that we try to avoid writing tests as much as possible rather than we write as many tests as required. I like the latter (your words) but am a bit dismayed by the former. It says to me that we don't really care about testing. I know that's not the intent but that's the message I'm getting.

Reply:
I think I prefer the way its currently worded.